### ISSUE TITLE
Running evaluation on FF++,DFDC,Celeb-DF

### ISSUE THREAD

**Issue by sharadchandakacherla** [yoctta/multiple-attention#25]
@yoctta  please help us out, 

1. I want to run the code in evaluation mode, however there is no celeb.json , i'm guessing this is for Celeb-DF-v1. Can you please outline steps how to evaluate the code?
2. Also could you please the exact file structure of your dataset directory and how to populate it?
3. There is no folder called runs with any config.pkl file.

P.S. : I just wan't to reload the shared weights and run inference on the test sets

Thanks

**Comment by sharadchandakacherla** [comment]
@yoctta  So i made dirs and trainconfig to generate the config.pkl file, and then was able to load the weights into EfficientNet-b4 from the .pth , but not sure how to get the preprocessing running  as im not sure how to get retina face running,
should I use the `pip install retina-face` or use https://github.com/biubug6/Pytorch_Retinaface?

**Comment by sharadchandakacherla** [comment]
closing the issue, was able to run evaluation successfully.

**Comment by DatDaiGia** [comment]
@sharadchandakacherla Could you please share how you ran the evaluation successfully? I am getting stuck in loading pre-trained weights. 
Many thanks in advance!
